src/macros/middleware.rs

```diff
@@ -33,7 +34,7 @@
 ///     format!("{}", visits.fetch_add(1, Ordering::Relaxed))
 /// });
 ///
-/// server.listen("127.0.0.1:6767").unwrap();
+/// server.listen("127.0.0.1:6767").await.unwrap();
```

Comment:
I don't love the `unwrap` but it seems to be the dominant idiom for async returns.  Not a blocker, just a general comment.